doc-content/enterprise-only/employee-rostering/optashift-ER-overview-source-con.adoc

```diff
@@ -21,26 +21,17 @@ The modules contain a number of files, including Java classes. This document lis
-** `*RestService.java` interfaces define the REST API. Both the server and the client application separately define implementations of these interfaces.
-
-* `employee-rostering-shared-gwt` module: Contains some of the classes required for the GWT client application.
-
-* `employee-rostering-webapp` module: Contains the HTML and other files necessary for building the entire application (client and server).
+* `employee-rostering-frontend` module: Contains the client application with the user interface, developed in React.
+
+* `employee-rostering-backend` module: Contains the server application that uses {PLANNER} to perform the rostering calculation.
+** `src/main/java/org.optaweb.employeerostering.service.roster/rosterGenerator.java` class: Generates random input data for demonstration and testing purposes. If you change the required input data, change the generator accordingly.
+** `src/main/java/org.optaweb.employeerostering.domain.employee/EmployeeAvailability.java`: Defines availability information for an employee. For every time slot an employee can be unavailable, available, or it can be designated a preferred time slot for the employee.
+** `src/main/java/org.optaweb.employeerostering.domain.employee/Employee.java`: Defines an employee. An employee has a name, a list of skills, and works under a contract. Skills are represented by skill objects.
+** `src/main/java/org.optaweb.employeerostering.domain.roster/Roster.java`: Defines the calculated rostering information.
+** `src/main/java/org.optaweb.employeerostering.domain.shift/Shift.java`: Defines a shift to which an employee can be assigned. A shift is defined by a time slot and a spot. For example, in a diner there could be a shift in the *Kitchen* spot for the February 20 8AM-4PM time slot. Multiple shifts can be defined for a given spot and time slot. In this case, multiple employees are required for this spot and time slot.
+** `src/main/java/org.optaweb.employeerostering.domain.skill/Skill.java`: Defines a skill that an employee can have.
+** `src/main/java/org.optaweb.employeerostering.domain.spot/Spot.java`: Defines a spot where employees can be placed. For example,  a `Kitchen` can be a spot.
+** `src/main/java/org.optaweb.employeerostering.domain.contract/Contract.java`:  Defines a contract that sets limits on work time for an employee in various time periods.
+** `src/main/java/org.optaweb.employeerostering.domain.tenant/Tenant.java`: Defines a tenant. Each tenant represents an independent set of data. Changes in the data for one tenant do not affect any other tenants.
+** `*View.java` classes related to domain objects do define value sets that are calculated from other information; the client application can read these values through the REST API, but not write them.
+** `*Service.java` interfaces located in the service package define the REST API. Both the server and the client application separately define implementations of these interfaces.
```

Comment:
For these last two (and I assume the asterisk is a wildcard), I think we should structure them consistently with all the ones above, like "`*View.java`: Classes that are related to ... and that define value ...", and same for the next one. And I'd reword "do define" to just define, and say "... other information. The client ..." with no semicolon. Clearer, and more consistent with the next one anyway.